app/src/main/java/at/bitfire/icsdroid/Settings.kt

```diff
@@ -6,44 +6,37 @@ package at.bitfire.icsdroid
 
 import android.content.Context
 import android.content.SharedPreferences
+import androidx.datastore.preferences.core.booleanPreferencesKey
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.longPreferencesKey
 import androidx.lifecycle.LiveData
 import kotlinx.coroutines.channels.awaitClose
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.single
+import kotlinx.coroutines.runBlocking
 
 class Settings(context: Context) {
 
     companion object {
-        private const val FORCE_DARK_MODE = "forceDarkMode"
-    }
-
+        @Deprecated("Use DataStore")
+        const val FORCE_DARK_MODE = "forceDarkMode"
 
-    private val prefs: SharedPreferences = context.getSharedPreferences("icsx5", 0)
+        val ForceDarkMode = booleanPreferencesKey("forceDarkMode")
 
-    fun forceDarkMode(): Boolean = prefs.getBoolean(FORCE_DARK_MODE, false)
+        val PrefNextReminder = longPreferencesKey("nextDonationReminder")
```

Comment:
Any reason the variable name is capitalized?

Reply:
[Same](https://github.com/bitfireAT/icsx5/pull/371#discussion_r1698469518).